main.tf

```diff
@@ -51,13 +51,24 @@ resource "ibm_database" "postgresql_db" {
   remote_leader_id                     = var.remote_leader_crn
   version                              = var.pg_version
   tags                                 = var.resource_tags
+  adminpassword                        = var.admin_pass
   service_endpoints                    = var.service_endpoints
   configuration                        = var.configuration != null ? jsonencode(var.configuration) : null
   key_protect_key                      = var.kms_key_crn
   backup_encryption_key_crn            = local.backup_encryption_key_crn
   point_in_time_recovery_deployment_id = var.pitr_id
   point_in_time_recovery_time          = var.pitr_time
 
+  dynamic "users" {
+    for_each = nonsensitive(var.users != null ? var.users : [])
```

Comment:
was there. reason for using `nonsensitive` here? the users variable has to be treated as sensitive as it will contain a password field

Reply:
cannot loop against sensitive values